package.json

```diff
@@ -44,6 +44,6 @@
     "postcss": "^8.5.3",
     "prettier": "^3.5.3",
     "tailwindcss": "^4.0.13",
-    "vite": "^6.2.0"
+    "vite": "^6.2.2"
```

Comment:
@lynar13 You could have asked why this upgrade of the Vite version? It is not necessary.

Note: This is a feedforward. As a QA, you don't just accept every change developers introduce.